### ISSUE TITLE
v0.3.2 SHOW CREATE TABLE doesn't fetch values

### ISSUE THREAD

**Issue by Barsoomx** [ClickHouse/clickhouse-connect#61]
Since v0.3.2, SHOW CREATE TABLE doesn't fetch any values via dbt load_result()

![image](https://user-images.githubusercontent.com/54457590/197160151-e46da2f3-b1fc-4d4e-82be-5fe632140b30.png)

Steps to reproduce:

Macros:
```
{% macro show_create_table2(relation) %}
    {% call statement('show_create_table', fetch_result=True) %}
        SELECT
            create_table_query AS ddl
        FROM system.tables
        WHERE
            name = '{{ relation.identifier }}' AND
            database = '{{ relation.schema }}'
    {% endcall %}

    {% do return(load_result('show_create_table').table) %}
{% endmacro %}


{% macro show_create_table3(relation) %}
    {% call statement('show_create_table', fetch_result=True) %}
        SHOW CREATE TABLE {{ relation.schema }}.{{ relation.identifier }}
    {% endcall %}

    {% do return(load_result('show_create_table').table) %}
{% endmacro %}
```

then in python console:

```
import os
import time
from clickhouse_connect.driver import HttpClient
from dbt.adapters.clickhouse import ClickHouseAdapter, ClickHouseRelation
from dbt.adapters.factory import get_adapter_by_type
from dbt.contracts.connection import Connection
from dbt.contracts.relation import Path
from dbt.lib import get_dbt_config
from dbt.tests.util import get_connection
config = get_dbt_config('/usr/app/')
ch_adapter = ClickHouseAdapter(config)
adapter = get_adapter_by_type(ch_adapter.type())
with get_connection(adapter) as conn:
    rel=adapter.get_relation(None, 'internal_analytics_src', 'src_dsas_datasources')
t1=adapter.execute_macro('show_create_table2', kwargs={"relation":rel})
t2=adapter.execute_macro('show_create_table3', kwargs={"relation":rel})
```

t2 is empty here, should be a table
![image](https://user-images.githubusercontent.com/54457590/197160809-6255f120-7128-402f-a30f-a15126929b3a.png)

I assume this is due to `Handle queries that are actually commands, simply and clean up tests`

But this command should load a table value, and it's counter-intuitive why it doesn't now.

**Comment by Barsoomx** [comment]
tagging @genzgd, dbt relevant)

**Comment by genzgd** [comment]
@Barsoomx This should be fixed in 0.3.3.

I'm curious as to how this is dbt relevant.  I don't see any use of SHOW CREATE TABLE in DBT, am I missing something?

**Comment by Barsoomx** [comment]
No, I thought it's relevant because of load_result. The macro I showed you is in dbt, albeit it's mine, sorry for the confusion, there's no such code in base dbt
